uniflow/flow/extract/extract_pdf_flow.py

```diff
@@ -33,6 +34,10 @@ def __init__(
             ),
         )
         self._process_pdf_op = ProcessPDFOp(name="process_pdf_op")
+        if splitter:
+            self._split_op = SplitterOpsFactory.get(splitter)
+        else:
+            self._split_op = None
```

Comment:
I would suggest to always assume there will be a splitter for pdf flow for now. It is good to have this if-else case, but it can introduce extra complexity and in the long term cause inconsistency from with and without splitter pdf extractor. in this case, you should use a default splitter with for example paragraph splitter and allow people to configure it to other splitter such as markdown. 

In the future, if there is a request regarding without splitter, we can think about bring it back. I would like to keep the code as simple as possible for now.